### DISCUSSION TITLE
FluxNews - Author Posts page

### DISCUSSION THREAD

**Discussion by ramoooi** [inspireui/support#5524] [New Feature/Ideas Requests]
It would be better when you clicked on the post author name would go to page which has all the posts for the selected author

and with a search engine to search for a post by specific author

same as follow:

https://prnt.sc/1pjLJODkVZCV

**Comment by batustun** [comment]
Definitely, authors list / authors widget / authors post page is a must for a Wordpress Blog app. I was expecting this feature actually when I purchased it. But I couldnt find. I think it should be adding. Thanks.
